tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -2112,6 +2112,18 @@ restapi/test_restapi.py:
     conditions:
       - "asic_type not in ['mellanox']"
 
+restapi/test_restapi.py::test_create_interface:
+  xfail:
+    reason: "Skipped due to community issue https://github.com/sonic-net/sonic-mgmt/issues/17997 on SN5 devices"
+    conditions:
+      - "https://github.com/sonic-net/sonic-mgmt/issues/17997 and 'SN5' in hwsku"
```

Comment:
hi @SavchukRomanLv , are these tests still failing? we are not seeing this test failures.

Reply:
HI @r12f  yes, it is still failing in our 202412 regression

Reply:
@r12f  - could it be approved by you?

Reply:
gotcha! thanks for confirming, @nhe-NV